include/wlr/types/wlr_output.h

```diff
@@ -192,6 +192,12 @@ void wlr_output_effective_resolution(struct wlr_output *output,
  * unknown. This is useful for damage tracking.
  */
 bool wlr_output_make_current(struct wlr_output *output, int *buffer_age);
+/**
+ * Get the preferred format for reading pixels.
+ * This also makes the output rendering context current.
```

Comment:
Hmm, I don't want users to start relying on this implementation detail, so I'd rather not document it, or say something like "this function can change the current rendering context".